constants.py

```diff
@@ -27,6 +27,7 @@
 
 google_url = "https://www.google.com/search?q="
 google_api_url = "https://www.googleapis.com/books/v1/volumes"
+google_image_url = "https://books.google.com/books/publisher/content/images/frontcover"

```

Comment:
Perfect use case where we can write simple unit tests. To make sure these API urls 

Here's a quick GPT unit test:

```python
import unittest
import requests
from PIL import Image
from io import BytesIO

class TestGoogleImageUrl(unittest.TestCase):
    def setUp(self):
        self.google_image_url = "https://books.google.com/books/publisher/content/images/frontcover"

    def test_image_url_exists_and_is_valid(self):
        response = requests.get(self.google_image_url)

        # Check that the request was successful (HTTP 200 OK)
        self.assertEqual(response.status_code, 200, "Failed to fetch image from URL.")

        # Check if the content-type is an image
        self.assertTrue("image" in response.headers["Content-Type"], "The URL does not return an image.")

        # Optionally, validate the image using PIL
        try:
            Image.open(BytesIO(response.content))
        except IOError:
            self.fail("The URL does not return a valid image.")

if __name__ == "__main__":
    unittest.main()
```

Thinking we could have this run periodically to ensure apis are working and urls haven't changed (for enrichment etc). Maybe a daily cron job with an email or telegram notification. And later we could even have a simple web interface we can visit like mercury.analogue.com/status that would show use the status of each test. Sort of like https://status.openai.com/